v3/src/components/graph/utilities/graph-utils.ts

```diff
@@ -382,7 +382,7 @@ export interface ISetPointSelection {
 export interface ISetPointCoordinates {
   anchor?: Point
   dataset?: IDataSet
-  pixiPointsRef: IPixiPointsRef
+  pixiPoints: PixiPoints
```

Comment:
```suggestion
  pixiPoints?: PixiPoints
```
If you make this optional, then you shouldn't need to change as many callers. The code is already testing for existence in line 427 below.

Reply:
Yes, thanks 👍